index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <![endif]-->
+<html lang="fr">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+</head>
+
+<body>
```

Comment:
- [ ] Please fix the horizontal scroll on your website because horizontal scrolling especially for pages with lots of text and buttons to click, it may not be a good experience for users if users have to scroll horizontally to see the remaining content of the page so its best to avoid it in your website, Kindly fix this 👍🏿 

https://user-images.githubusercontent.com/69638013/159013350-2a71cf96-91e1-4faa-90d5-cc117d23c2e9.mp4